usr/include/include.mk

```diff
@@ -8,4 +8,5 @@ opencryptokiinclude_HEADERS =						\
 noinst_HEADERS +=							\
 	usr/include/apictl.h usr/include/local_types.h			\
 	usr/include/pkcs32.h usr/include/slotmgr.h usr/include/stdll.h	\
-	usr/include/events.h
+	usr/include/events.h usr/include/aix/asprintf.h usr/include/aix/endian.h \
+	usr/include/aix/err.h usr/include/aix/getopt.h usr/include/aix/secure_getenv.h
```

Comment:
I would say that `platform.h` should also be added to `noinst_HEADERS`. We don't want to / need to install `platform.h`.